app/models/user.rb

```diff
@@ -1,12 +1,18 @@
 class User < ActiveRecord::Base
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  mount_uploader :avatar, AvatarUploader
+
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
+  attr_accessible :avatar, :avatar_cache, :remove_avatar
+
+  validates_presence_of :avatar
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.